src/commands/infra/generate.ts

```diff
@@ -0,0 +1,190 @@
+import child_process from "child_process";
+import { getMaxListeners } from "cluster";
+import commander from "commander";
+import fs, { chmod } from "fs";
+import * as os from "os";
+import path from "path";
+import simpleGit from "simple-git/promise";
+import { logger } from "../../logger";
+
+const spkTemplatesPath = os.homedir() + "/.spk/templates";
```

Comment:
Will there ever be a use case where this will need to differ?  Can it be recreated with an existing cluster?

Reply:
Related to comment before around the cache folder. We can support this function, just needs to be built into `spk init`

Reply:
Resolved per above.